### DISCUSSION TITLE
6.1 简体汉化出错

### DISCUSSION THREAD

**Discussion by VonEinzbern14** [GpointChen/FFXIVChnTextPatch-GP#133] [General]
我用了rawexd-zhcn-0413.rar 中的文件进行替换并汉化。
但是开启游戏的时候出现了文件错误，请问是由于更新导致的么？
另外中国服的0a0000三个文档目前也是无法进行汉化的。

**Comment by GpointChen** [comment]
應已修正於v2.59

另外，6.0和6.1檔案結構有差異，所以用中國服0a0000會無法漢化。
